manifest.toml

```
@@ -0,0 +1,52 @@
+packaging_format = 2
+
+id = "wemawema"
+name = "WemaWema"
+description.en = "\"WE MAKE PORN\" meme generator but it can do more"
+description.fr = "Générateur de mème « WE MAKE PORN » mais il peut faire bien plus"
+
+version = "26.1~ynh4"
+
+maintainers = ["Luc Didry"]
```

```suggestion
maintainers = []
```

I don't think he's actively maintaining this app (at least the yunohost package i mean) so let's remove it ...